### ISSUE TITLE
Datatable headerRow expanded attribute

### ISSUE THREAD

**Issue by PlassJ** [primefaces/primefaces#2443]
While using the Datatable with expandableRowGroups, it would be really nice to control the togglestate of each headerRow.

I would like to suggest an expanded attribute for headerRow, see sample XHTML.

Sample XHTML:
``` xhtml
<p:dataTable var="car" value="#{dtRowGroupView.cars}" sortBy="#{car.brand}" expandableRowGroups="true">
        <p:headerRow expanded="#{car.brand eq 'BMW'}" >
            <p:column colspan="3">
                <h:outputText value="#{car.brand}" />
            </p:column>
        </p:headerRow>
        <p:column headerText="Year">
            <h:outputText value="#{car.year}" />
        </p:column>
</p:dataTable>`
```

**Comment by Dougnlizt** [comment]
@PlassJ , I see you closed this request.  Any particular reason why?  I thought it was a good suggestion, and I'd like to see it added as well.

**Comment by PlassJ** [comment]
@Dougnlizt , oh dear .... this is nearly 2 years old .... @tandraschko did you know something about this?

**Comment by melloware** [comment]
fixed for 9.0: https://github.com/primefaces/primefaces/issues/6325
